crates/uv-cli/src/lib.rs

```diff
@@ -2181,15 +2194,13 @@ pub struct RunArgs {
 #[derive(Args)]
 #[allow(clippy::struct_excessive_bools)]
 pub struct SyncArgs {
-    /// Include optional dependencies from the extra group name; may be provided more than once.
+    /// Include optional dependencies from the extra group name.
     ///
-    /// Only applies to `pyproject.toml`, `setup.py`, and `setup.cfg` sources.
+    /// May be provided more than once.
```

Comment:
I feel like this got removed from some other commands in previous PRs?

Reply:
I've usually moved it out of the first line but retained it in general. I can audit again at the end.